FH-3785 Add initial dashboard for sync server

aidenkeating opened the pull request:
Add an initial dashboard for sync server.

This is the start of the sync server dashboard. It's a basic
representation of the /sys/info/stats response JSON using cards.

Some things that will probably want to be added in the future:

* Charts/graphs instead of cards for some metrics, timings etc
* Navigation bar when adding the config screen
* Better support for errors/missing data

Ensure you have a server that has been synced against at least once so that metrics is enabled. Go to /dashboard.html

For macs CPU will never work.

<img width="1409" alt="screen shot 2017-08-22 at 10 03 58" src="https://user-images.githubusercontent.com/8698527/29557626-9c1edba0-8721-11e7-9643-9b00ff959907.png">

@david-martin Would you mind taking a look? Wasn't sure how far to go with this.

Commit:
FH-3785 Add initial dashboard for sync server

Add an initial dashboard for sync server.

This is the start of the sync server dashboard. It's a basic
representation of the /sys/info/stats response JSON using cards.

Some things that will probably want to be added in the future:

* Charts/graphs instead of cards for some metrics, timings etc
* Navigation bar when adding the config screen

david-martin commented:
@aidenkeating I'm good with this as an initial dashboard view of stats.
However, it would be good to get this fix for the `fh-sync` module in before merging
https://github.com/feedhenry/fh-sync/pull/35

Commit:
FH-3785 Update fh-sync dependency

Currently, using fh-sync, the metrics that are displayed do not
update. In v1.0.11 the way metrics are retrieved has changed,
this results in metrics being updated regularly and them being the
current value. Not the least recent recorded value.

aidenkeating commented:
@david-martin Updated the fh-sync dependency, everything is updating more regularly now. I think this is ready to merge.
